### PR TITLE
Fix  DefaultEndpointTestCase

### PR DESCRIPTION
**Pull request by himasha**

### CODE REVIEW

**Comment by riyafa** [comment]
I am only curious. What does this fix change in the test case?

**Comment by himasha** [comment]
@riyafa  According to failed logs, even though the endpoint is suspended in the first call there are occurrences where the response of the first call is sent before the second call is invoked. In this case, Assert.assertNull(resp) fails because we are referring to the same variable value.

File: integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/DefaultEndpointTestCase.java [hunk 1]
```diff
@@ -124,27 +124,28 @@ public void testSendingToDefaultEndpointWithSuspension()
...
-        Assert.assertNull(response);
+        Assert.assertNull(response1);
         //Increasing wait time than suspendDuration value
         Thread.sleep(15000);
-        response = axis2Client.sendSimpleStockQuoteRequest(getProxyServiceURLHttp("defaultEndPoint_Config_Reg"),
+        response2 = axis2Client.sendSimpleStockQuoteRequest(getProxyServiceURLHttp("defaultEndPoint_Config_Reg"),
                                                            "http://localhost:9001/services/SimpleStockQuoteService", "WSO2");
-        Assert.assertNotNull(response);
-        Assert.assertTrue(response.toString().contains("WSO2 Company"));
+        Assert.assertNotNull(response2);
```
Comment (sdkottegoda) [inline integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/DefaultEndpointTestCase.java]: Shall we print the response when the assertion fails?

File: integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/DefaultEndpointTestCase.java [hunk 2]
```diff
@@ -124,27 +124,28 @@ public void testSendingToDefaultEndpointWithSuspension()
...
+        Assert.assertTrue(response2.toString().contains("WSO2 Company"));
```
Comment (sdkottegoda) [inline integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/DefaultEndpointTestCase.java]: Same as above?

File: integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/DefaultEndpointTestCase.java (hunk 1 above)
Comment (himasha) [reply integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/DefaultEndpointTestCase.java]: Added print for assertNull() locations.

**Pull request merged**